configs/defs/fallback.textproto

```diff
@@ -10187,3 +12373,710 @@ benchmark_configs: {
+      "        }"
+      "        match_criteria: {"
+      "          filter_regex: \".*enforcing=[0-9]+.*\""
+      "          expected_regex: \".*enforcing=0.*\""
+      "        }"
+      "      }"
+      "    }"
+      " }"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/default/grub\"}}"
+      "      existence:{should_exist:false}"
+      "    }"
+      " }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "selinux-enforcing-fallback"
+  compliance_note: {
+    version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    title: "Ensure the SELinux state is enforcing"
+    description:
+      "Set SELinux to enable when the system is booted."
+    rationale:
+      "SELinux must be enabled at boot time to ensure that " 
+      "the controls it provides are in effect at all times."
+    remediation:
+      "Edit the /etc/selinux/config file to set the SELINUX parameter: \n"
+      "```SELINUX=enforcing```"
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/selinux/config\"}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*SELINUX\\\\s*=\\\\s*\\\\w+.*\""
+      "          expected_regex: \"\\\\s*SELINUX\\\\s*=\\\\s*enforcing.*\""
+      "        }"
+      "      }"
+      "    }"
+      " }"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/selinux/config\"}}"
+      "      existence:{should_exist:false}"
+      "    }"
+      " }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "selinux-policy-configured-fallback"
+  compliance_note: {
+    version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    title: "Ensure SELinux policy is configured"
+    description:
+      "Configure SELinux to meet or exceed the default targeted policy, "
+      "which constrains daemons and system software only."
+    rationale:
+      "Security configuration requirements vary from site to site. "
+      "Some sites may mandate a policy that is stricter than the default policy, "
+      "which is perfectly acceptable. "
+      "This item is intended to ensure that at least the default recommendations are met."
+    remediation:
+      "Edit the /etc/selinux/config file to set the SELINUXTYPE parameter: \n"
+      "```SELINUXTYPE=targeted```\n"
+      "Notes: If your organization requires stricter policies, "
+      "ensure that they are set in the /etc/selinux/config file."
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
```

Comment:
check_alternative for /etc/selinux/config not existing here too

Reply:
You can verify if all use cases are covered by running Localtoast with chroot set to an empty directory with e.g. `./localtoast [...] --chroot=/tmp/dir` and checking that no benchmarks are non-compliant